### ISSUE TITLE
DataCamp Track: Machine Learning Scientist in Python

### ISSUE THREAD

**Issue by dannydore** [dannydore/risk-tech-training#22]
[DataCamp Track: Machine Learning Scientist in Python](https://app.datacamp.com/learn/career-tracks/machine-learning-scientist-with-python)
---
- [ ] Supervised Learning with scikit-learn
- [ ] Project: Predictive Modeling for Agriculture
- [ ] Unsupervised Learning in Python
- [ ] Project: Clustering Antarctic Penguin Species
- [ ] Linear Classifiers in Python
- [ ] Machine Learning with Tree-Based Models in Python
- [ ] Project: Predicting Movie Rental Durations
- [ ] Extreme Gradient Boosting with XGBoost
- [ ] Cluster Analysis in Python
- [ ] Dimensionality Reduction in Python
- [ ] Preprocessing for Machine Learning in Python
- [ ] Machine Learning for Time Series Data in Python
- [ ] Feature Engineering for Machine Learning in Python
- [ ] Model Validation in Python
- [ ] Hyperparameter Tuning in Python
- [ ] Skill Assessment: Machine Learning Fundamentals in Python
- [ ] Introduction to Natural Language Processing in Python
- [ ] Natural Language Processing with spaCY
- [ ] Feature Engineering for NLP in Python
- [ ] Introduction to Deep Learning with PyTorch
- [ ] Intermediate Deep Learning with PyTorch
- [ ] Image Processing in Python
- [ ] Introduction to PySpark
- [ ] Machine Learning with PySpark
- [ ] Winning a Kaggle Competition in Python
